src/app/shared/post-capture-card/post-capture-card.component.scss

```diff
@@ -36,8 +38,19 @@ mat-card-header {
 }
 
 mat-card-actions {
-  padding-right: 16px;
+  width: 100vw;
+  padding-right: 8px;
   padding-left: 16px;
+
+  .all-btn {
```

Comment:
Do not use abbreviation. Use `.view-all-button` or `.flat` instead.

Actually, this button style can be simply achieved by applying `mat-flat-button color="primary"` properties from Angular Material library.